syntax = "proto3";

package fraud_detection;


service FraudDetection {
    rpc FraudDetection (FraudDetectionRequest) returns (FraudDetectionResponse) {}
}

message FraudDetectionRequest {
    string number = 1;
    string expirationDate = 2;
}

message FraudDetectionResponse {
    bool is_fraud = 1;
    string reason = 2;
}
